maintainers/maintainer-list.nix

```
@@ -24823,6 +24823,11 @@
     githubId = 24979302;
     name = "Vladimír Zahradník";
   };
+  wgrav = {
+    name = "wgrav";
+    github = "wgrav01";
+    githubId = 63617338;
+    email = "wgrav01@protonmail.com";
```

```suggestion
    email = "wgrav01@protonmail.com";
  };
```
Needs to be sorted alphabetically.